cypress/e2e/certificate-provider/confirm-your-details.cy.js

```diff
@@ -1,7 +1,7 @@
 describe('Confirm your details', () => {
   describe('shows details for', () => {
     it('lay certificate providers', () => {
-      cy.visit('/fixtures/certificate-provider?redirect=/enter-date-of-birth');
+      cy.visit('/fixtures/certificate-provider?redirect=/enter-date-of-birth&sendLpaToLpaStore=1');
```

Comment:
if you remove this the test still passes, so does it need to be sent?

Reply:
Hmmm interesting - I was getting the details back from the LPA store rather than the donor-entered details but this was before I'd merged in main a couple of times. All looks fine, as you pointed out, so I'll drop this.